src/nupic/utils/LoggingException.cpp

```diff
@@ -34,7 +34,11 @@ LoggingException::~LoggingException() throw()
   if (!alreadyLogged_) {
     // Let LogItem do the work for us. This code is a bit complex
     // because LogItem was designed to be used from a logging macro
+#if defined(NTA_OS_WINDOWS) && defined(NTA_COMPILER_MSVC) && defined(NDEBUG)
+    auto li = new LogItem(filename_.c_str(), lineno_, LogItem::info);
```

Comment:
Here's a working build using LogItem::info [1]. Reverting back to just use LogItem::error [2]. I thought it might just be a RandomTest, but deleting two test cases still fails the build (but not the tests) [3].

This does need putting back to only use LogItem:error. Through scenario 2 I need to work out why the build process returns a fail to AppVeyor. Finding that, hopefully we can convince AppVeyor that the build is ok. On top of this is also a Cap'n'Proto test fail in more recent code relating to a EOF problem that we're looking at. So a few test issues to resolve

See line 950 for test_htm passing, and line 1310 for unit_test.
1 https://ci.appveyor.com/project/rcrowder/nupic-core/build/1.0.129
2 https://ci.appveyor.com/project/rcrowder/nupic-core/build/1.0.130
3 https://ci.appveyor.com/project/rcrowder/nupic-core/build/1.0.131